javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -245,6 +245,21 @@ class Expr extends @expr, ExprOrStmt, ExprOrType, AST::ValueNode {
       ctx.(ConditionalExpr).inNullSensitiveContext()
     )
   }
+
+  /**
+   * Gets the data-flow node where exceptional data-flow will flow if this expression
```

Comment:
The phrase "data-flow will flow" a bit unfortunate.

How about "where exceptions thrown by this expression will propagate"